dev/com.ibm.ws.opentracing.1.1/src/com/ibm/ws/opentracing/OpentracingJaxRsEMCallbackImpl.java

```diff
@@ -39,7 +39,8 @@ public OpentracingJaxRsEMCallbackImpl() {
 
     @Override
     public Map<String,Object> onDefaultMappedException(Throwable t, int statusCode, ResourceInfo resourceInfo ) {
-        Tr.warning(tc, "OPENTRACING_UNHANDLED_JAXRS_EXCEPTION", t);
+        if (!t.getMessage().contains("HTTP 404 Not Found"))
```

Comment:
What would be the value of `statusCode`?  If it is 404, it would be better to check that than checking for a message.